src/frontend/get-github-configuration.ts

```diff
@@ -65,41 +69,33 @@ async function getInstallationsWithAdmin(log: Logger,
 			type: installation.target_type
 		});
 
-		const githubClient = new GitHubClient(getCloudInstallationId(installation.id), log);
-		const numberOfReposPromise = githubClient.getNumberOfReposForInstallation();
-
-		const [admin, numberOfRepos] = await Promise.all([checkAdmin, numberOfReposPromise]);
+		try {
+			const githubClient = new GitHubClient(getCloudInstallationId(installation.id), log);
+			const numberOfReposPromise = githubClient.getNumberOfReposForInstallation();
+			const [admin, numberOfRepos] = await Promise.all([checkAdmin, numberOfReposPromise]);
 
-		log.info("Number of repos in the org received via GraphQL: " + numberOfRepos);
+			log.info("Number of repos in the org received via GraphQL: " + numberOfRepos);
 
-		installationsWithAdmin.push({
-			...installation,
-			numberOfRepos: numberOfRepos || 0,
-			admin
-		});
+			installationsWithAdmin.push({
+				...installation,
+				numberOfRepos: numberOfRepos || 0,
+				admin
+			});
+		} catch (err) {
+			log.warn({err, installationId: installation.id, org: installation.account.login}, "Cannot check admin or get number of repos per org")
```

Comment:
Am I correctly understand that adding this try-catch block it is the main change in this PR and everything else is merely a refactoring?

Reply:
How would it look in UI? We won't display any message aren't we?

Reply:
That's correct.  The refactoring was needed though as we needed a way to separate installations that worked vs those that didn't, and then decide if we want to uninstall those that didn't.

No UI, at this point we're just preventing the page from crashing altogether.  I'll have another PR where we can show the ones that failed with a message to the user that I need to talk to Jade about.